region/norfair/west/Crocomire Speedway.json

```diff
@@ -736,20 +1265,83 @@
-          "enemy": "Cacatac",
-          "type": "spike",
-          "hits": 1
-        }},
-        {"shinespark": {"frames": 92, "excessFrames": 10}},
-        {"heatFrames": 900}
+        {"or": [
+          {"enemyDamage": {
+            "enemy": "Cacatac",
+            "type": "spike",
+            "hits": 1
+          }},
+          "canBePatient"
+        ]},
+        {"or": [
+          {"shinespark": {"frames": 88, "excessFrames": 11}},
+          {"and": [
+            "h_heatProof",
+            "canControlShinesparkEnd",
+            {"shinespark": {"frames": 88, "excessFrames": 44}}
+          ]}  
+        ]},
+        {"heatFrames": 825},
+        {"or": [
+          "canSpeedball",
+          {"heatFrames": 15}
+        ]}
       ],
       "clearsObstacles": ["A"],
       "note": [
-        "The timer is tight, so run through the Cacatacs and mid-air spark left through the speed blocks.",
+        "Either jump over a Cacatac or tank a spike hit.",
+        "Perform a mid-air spark through the speed blocks.",
         "Then run to the right and back to get speed to go through the rest.",
-        "The shinespark expects to kill the crumble bridge pirate, to be safe."
+        "If performing the spark with low energy, there is a risk of ending the spark above the spikes or in front of the Pirate;",
+        "to be safe, assuming Samus has heat protection, begin the shinespark with between 73 and 88 energy, or at least 106 energy;",
+        "with a buffered crumble jump to the right, up to 94 energy can work."
+      ],
+      "devNote": [
+        "The canBePatient represents retrying until RNG works out to be able to jump over one of the Cacatacs without taking a hit.",
+        "There is a possibility of getting a lucky drop from one of the two Pirates;",
+        "but since it is low probability, we don't model it.",
+        "The speedball exit is technically not a valid way to leave normally (e.g. in case the next room is heated and Samus needs to jump);",
+        "but if necessary, it is possible to unmorph near the transition."
+      ]
+    },
+    {
+      "link": [5, 2],
+      "name": "Reverse Spark With Reserve Abuse",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 5,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        {"notable": "Reverse Spark"},
+        "canShinechargeMovementTricky",
+        "canTrickyJump",
+        "canControlShinesparkEnd",
+        "canPauseAbuse",
+        {"resourceAvailable": [{"type": "RegularEnergy", "count": 99}]},
+        {"resourceConsumed": [{"type": "ReserveEnergy", "count": 100}]},
+        {"or": [
+          "canPreciseReserveRefill",
+          {"resourceConsumed": [{"type": "ReserveEnergy", "count": 15}]}          
+        ]},
+        {"resourceAtMost": [{"type": "RegularEnergy", "count": 1}]}
+      ],
+      "clearsObstacles": ["A"],
+      "note": [
+        "Either jump over a Cacatac or tank a spike hit.",
```

Comment:
I couldn't survive the trip taking a spike hit

Reply:
My problem was with the 11 energy pauses.  I could probably eventually do it.

If you farm the cac, this becomes fairly easy, if you hit the shinespark.
full supers, missile/grapple/plasma/wave+spazer

Reply:
With this change, the strat moves down to Expert in the case where the Cac can be farmed for a big energy. I think it's probably fair since it's just regular pause abuse at that point, nothing precise. And even though it's a long sequence of them, the pause abuse here is straightforward since you're not doing any other movement, only holding dash and left.

Reply:
I'm not sure I understand, is there something you want changed here? This is currently a Hard notable but the strats are in Expert by the tech requirements (or Extreme if coming from top-right with no ability to farm the Cac).

Positioning for the spark does seem like the most difficult part of the trick, and that mostly applies whether or not doing pause abuse (though the pause before wind-up does add another distraction). On the other hand, positioning vertically within a 3-tile space for a spark isn't something that we've been bumping higher than Expert in other places? Most cross-room sparks that require specifically "top" or "bottom" position could require that level of precision, but they would usually be in Very Hard based on `canShinechargeMovementComplex`, or sometimes even Hard if it doesn't need `canShinechargeMovementComplex`.

Reply:
I want pause abusing up the speedway to be a new notable.

`[5, 2] Reverse Spark With Reserve Abuse` does seem to get better with more practice, as does timing the repauses.  But getting close to 11 isn't so simple.  I think it fits in Expert, but is directly harder than `[5, 2] Reverse Spark`.
And it needs to be clearer that 0+1 tanks is enough to make this run.